composables/masto/statusDrafts.ts

```diff
@@ -30,7 +30,7 @@ export function getDefaultDraft(options: Partial<Mutable<mastodon.v1.CreateStatu
     params: {
       status: status || '',
       inReplyToId,
-      visibility: visibility || 'public',
+      visibility: currentUser.value?.account.source.privacy || visibility || 'public',
```

Comment:
There was another regression here, when you answer to a private post, the reply should be private by default.

The default privacy only should be taken into account if it makes the post more private than the replying to post

Reply:
Thanks. Can you check the updated logic?